multiversx_sdk_cli/message.py

```diff
@@ -0,0 +1,20 @@
+from typing import Dict
+
+from multiversx_sdk_cli.accounts import Account
+
+
+class SignableMessage:
+    def __init__(self, message: str, account: Account) -> None:
+        self.message = message
+        self.account = account
+
+    def sign(self) -> None:
+        hex_signature = self.account.sign_message(self.message.encode())
+        self.signature = bytes.fromhex(hex_signature)
```

Comment:
The "signable" message now becomes a "signed message".

Alternatively, could be `SignedMessage(message, account)` and also do `self.signature = ...` in the constructor. Not an actual improvement, though. 

Or have this file renamed to `sign_verify.py` and have a `sign_message(message, account)` return a simple DTO called `SignedMessage` (also defined in this file), that has the `to_dictionary()` method, as well. Then, some parts of the verify logic can be moved here, too (for symmetry).

Optional, can also stay as it is.

Reply:
Indeed, great suggestion!